whynotearth/BrowTricks
# 324 re-route to home after auth

# Issue Being Addressed

#324 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[ ] New Feature
[x] Update to Existing Feature
[ ] Other (state below)

# How to Test/Reproduce

login/signup and see redirect to tenant home is working.

-- 
Also fix privacy page.
